test/hack/resource-management/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/aws/karpenter/test/hack/cleanup
+module github.com/aws/karpenter/test/hack/resource-management
```

Comment:
There's a little write-up on naming go packages and modules: https://go.dev/blog/package-names. I'd agree that naming is hard here, but I think the second you are throwing hyphens and underscores into the mix, you could probably shorten or rename. 

Consider the following renames:
1. Module: `github.com/aws/karpenter/test/hack/resources`
2. Cleanup Script: `github.com/aws/karpenter/test/hack/resources/clean`
3. Count Script: `github.com/aws/karpenter/test/hack/resources/count`